tide_landing_page.install

```diff
@@ -1015,7 +1015,14 @@ function tide_landing_page_update_8025(&$sandbox) {
     $sandbox['processed']++;
   }
   $sandbox['#finished'] = ($sandbox['processed'] / $sandbox['total']);
```

Comment:
Would this still return the same response? I think it's trying to do something like ```5 / 0``` and its throwing a warning on that.

Reply:
if https://github.com/dpc-sdp/tide_landing_page/blob/develop/tide_landing_page.install#L995 equals 1, it will stop the the batch.

Reply:
I think the issue is that there should not have codes beneath https://github.com/dpc-sdp/tide_landing_page/pull/104/files#diff-1cb50536ef66dd94cde046ac1401db78L1017 . if it has code , which causes the error you encountered in content-together-vic-gov-au